libs/driver-adapters/src/conversion/mod.rs

```diff
@@ -13,3 +13,13 @@ pub(crate) mod sqlite;
 
 pub use js_arg::JSArg;
 pub use js_arg_type::{value_to_js_arg_type, JSArgType};
+
+/// A wrapper around `Option` that indicates that `None` should be treated as
+/// `undefined` in JavaScript.
+pub struct MaybeDefined<T>(pub Option<T>);
+
+impl<V> From<Option<V>> for MaybeDefined<V> {
+    fn from(value: Option<V>) -> Self {
+        Self(value)
+    }
+}
```

Comment:
Note from the call: `MaybeDefined` ensures that `Option::None` is consistently serialised into `undefined` with both `napi` and `wasm_bindgen`.